### DISCUSSION TITLE
Notification template to notify API consumers about new API versions

### DISCUSSION THREAD

**Discussion by AnRei123** [Azure/api-management-developer-portal#2556] [Feature request]
## Bug description

Currently, there is no automated mechanism in place to inform the subscriber of an API product about newly released API versions (breaking changes) in form of an email notification. We got the feedback that the consumer do not frequently open the portals to proactively look up changes. 

Our API consumers should always use the latest API version as soon as possible. After a new version has been created, we like to ensure that our API consumer are informed accordingly. This becomes especially important as the products can now release independently and as their are continuous releases where we couldn´t request our consumer to monthly query for new API versions in several production environments.

So we urgently need this notification template for this purpose as soon as possible.

## Expected behavior

A **notification template** with the name "**New API version for a subscribed API product has been made current**" in the APIM portal is available that can be customized and configured for each environment.
The configuration of the notification template allows to use parameters such as "**Name of the API product**", "**Name of the API version**" (incl. the version), "**list of associated Operations**" (as developers mostly remember the names of the operations) and "**API version details URL**" (link to the API descriptions and the changelogs of the affected API version).

Especially for our Sandbox and production environments, we urgently need this notification as soon as possible to minimize the maintenance efforts and to increase the consumer satisfaction - even in case of declaring a deprecation after a certain period of time after the new API version had been announced by email notification.

To control and restrict the amount of emails the portal user get, we would require the notification only for consumers that have subscribed to the associated API products.

An addition, it might be great to allow sorting the list of APIs for the latest API versions and filter them according to the consumers subscriptions. But for us this would not be sufficient as we would like to ensure that even consumers that are not frequently visiting the portal are aware of breaking changes for their subscribed API products.

We urgently need this missing notification template as soon as possible to keep our consumers happy and to be able to get our users to the latest versions as soon as possible. This allows us to earlier declare previous API versions have been deprecated. 

Would it be realistic and possible to get such a notification template within the next one or two months?

## Is your portal managed or self-hosted?

Managed

## Environment

 - Operating system: Windows 10
 - Browser: Chrome (latest)

**Comment by mikebudzynski** [comment]
@AnRei123 Although we don't provide this functionality as a single-click action, you can retrieve users who use a particular API from the built-in API reports and create a custom workflow to notify them about new releases (for example, you can use Azure Logic Apps). We'll note this Issue as a feature request.

**Comment by mikebudzynski** [comment]
Related to #583

**Comment by spaasis** [comment]
Is there any progress made on this? I was surprised to find this not supported out of the box

**Comment by Dayna-A** [comment]
Also wondering about this. We are looking for a solution for this without needing to build custom workarounds.
